app/javascript/src/experiment-solution.js

```diff
@@ -26,6 +28,13 @@ class ExperimentSolution {
     this._setupSplit();
   }
 
+  _checkWebsocketsCompatibility() {
+    if(!('WebSocket' in window) && !('MozWebSocket' in window)) {
```

Comment:
I got this snippet from https://stackoverflow.com/questions/20060320/what-is-the-best-way-to-detect-websocket-support-using-javascript.